ios/Mattermost/MattermostManaged.m

```diff
@@ -59,6 +60,15 @@ - (NSDictionary *)constantsToExport {
             });
 }
 
+RCT_EXPORT_METHOD(isHwKeyboardActive:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
+  if (@available(iOS 14.0, *)) {
```

Comment:
Do we also need a conditional for iPadOS?

Reply:
I thought about this too, but there doesn't seem to be an availability check for iPadOS:


<img width="400" alt="Screenshot 2022-08-13 at 11 14 47 AM" src="https://user-images.githubusercontent.com/24620401/184470495-b2c927ad-803a-4ee0-8219-8428f17ffab5.png">


From what I understand, checking for an iOS version also checks for the corresponding iPadOS version; ie., iOS 14.0 == iPadOS 14.0, etc.

Reply:
I think the idea is to also check for the idiom to be ipad, as gor ios based on the ticket this should not apply even if you connect the keyboard to the phone, the way this is done is applicable to all devices

Reply:
Interesting, just found out about `UI_USER_INTERFACE_IDIOM()`. I'll add an additional condition to only check for a keyboard on iPad.